lib/irb/cmd/pushws.rb

```diff
@@ -11,27 +11,38 @@
 #
 
 require_relative "nop"
-require_relative "../ext/workspaces"
```

Comment:
I understand why you may want to do this when you introduce `show_cmds`, but it's technically an independent feature. `show_cmds`/`category`/`description` alone has enough complexity, so I'd appreciate if you can separate a pull request for loading those commands more lazily.

Reply:
:+1: Can you remove the diff from this PR? They will hopefully not conflict with each other.

Reply:
Yeah sure. I'll rebase this branch after #472 is merged.